framework/source/class/qx/ui/form/core/VirtualDropDownList.js

```diff
@@ -326,6 +376,7 @@ qx.Class.define("qx.ui.form.core.VirtualDropDownList",
      * @param event {qx.event.type.Data} The change event.
      */
     _onChangeDelegate : function(event) {
+      "use strict";
```

Comment:
Please do not change to strict mode in the framework. This may have implications which lead to failing code. For example I'm not really sure if this works when using the generator, as in strict mode `arguments.callee` is not available anymore. So someone deriving from this class and using the `this.base(arguments` syntax to call the superclass method will fail with generator, as `this.base` relies on `arguments.callee` being available.

Reply:
This is only the case if you try to use `this.base` in a method that has strict mode; it doesn't matter if the superclass method has strict mode unless that superclass method also uses `this.base`. I will remove it for now.